### DISCUSSION TITLE
Tracking status change on issue on a Project

### DISCUSSION THREAD

**Discussion by simonebracaloni** [community/community#38270] [Projects and Issues]
We started using the new [Project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) on Github and we quite happy usign it.
Yet I cannot find a lot of (IMHO) useful features and i was wandering where is the best place to discuss about it.

Is ther a specific category on Github community?

I would love to have the possibility to track the status change of the issue in the project but I cannot figure out how to do so, people in my team are keeping moving issue from one status col to another but this is not tracked anywhere. 
Would it be possible to implemt something with GiHub Actions?

Thanks to everybody who want to contribute to this topic.

**Comment by ossanna16** [comment]
Hi there @simonebracaloni and welcome to our community! Thank you for the feedback 🙂

**Comment by queenofcorgis** [comment]
@simonebracaloni you've come to the right place - the Issues category is for feedback, bug reports, questions, and any other related conversations about GitHub Issues and Projects.

**Reply by simonebracaloni** [reply]
Thanks @queenofcorgis ! Do you know if it is possible to automatically add comment to the issue based on the activity done on the project (e.g. issue status is moved from "to do" to "in progress") ?
